tests/Feature/MyTests/NoteTest.php

```diff
@@ -0,0 +1,159 @@
+<?php
+
+namespace Tests\Feature;
+
+// use Illuminate\Foundation\Testing\RefreshDatabase;
+use App\Models\Note;
+use App\Models\User;
+use Egulias\EmailValidator\Parser\Comment;
+use Egulias\EmailValidator\Parser\CommentStrategy\LocalComment;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use PHPUnit\Framework\SkippedTest;
+use Tests\TestCase;
+
+/*
+    Tests will follow the AAA (Arange Act Assert) format
+*/
+
+class NoteTest extends TestCase
+{
+    use RefreshDatabase; //clears the database for each test run
+    protected $userOne, $userTwo;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+        // Create 2 test users, need 2 so we can test note viewing authentication
+        $userData = [];
+
+        for ($i = 1; $i <= 2; $i++) {
+            $userData[] = [
+                'name' => 'Test User #' . $i,
+                'email' => 'testemail' . $i . '@test.com',
+                'password' => bcrypt('pass123.'),
+            ];
+        }
+
+        User::factory()->createMany($userData);
+        $this->userOne = User::find(1); //assign userOne
+        $this->userTwo = User::find(2);//assign userTwo
```

Comment:
Thank you for pointing that out! I definitely see how this could be referencing other users if I only rely on their id. And if there was a lot of users I might not even know the users id that I'm referencing. In the future I should definitely reference their name or email like you said:
`$userOne = User::where('name', 'Test User #1')->first();`
`$userTwo = User::where('name', 'Test User #2')->first();`